contents/docs/data/identify.mdx

```diff
@@ -8,116 +8,41 @@ availability:
+<JSIdentifyHowItWorks/>
 
 ## Best practices when using `identify`
 
 ### 1. Call `identify` as soon as you're able to
 
-In your frontend, you should call `identify` as soon as you're able to. Typically, this is every time your app loads for the first time, or directly after your users log in. This ensures that events sent during your users' sessions are correctly associated with them. 
-
-You only need to call `identify` once per session.
+<JSIdentifyBestPractices1/>
 
 ### 2. Use unique strings for distinct IDs
 
-If two users have the same distinct ID, their data will be merged and they will be considered as one user in PostHog. Two common ways this can happen are:
-
-- Your logic for generating IDs does not generate sufficiently strong IDs and you can end up with a clash where 2 users have the same ID.
-- There's a bug, typo, or mistake in your code leading to most or all users being identified with generic IDs like `null`, `true`, or `distinctId`.
-
-PostHog also has built-in protections to stop the most common distinct ID mistakes. See the [FAQ](/docs/data/identify#what-happens-if-you-call-identify-or-alias-with-invalid-inputs) at the end of this page for more details.
+<JSIdentifyBestPractices2/>
 
 ### 3. Reset after logout
 
-If a user logs out on your frontend, you should call `reset` to unlink any future events made on that device with that user. 
-
-This is important if your users are sharing a computer, as otherwise all of those users are grouped together into a single user due to shared cookies between sessions. **We strongly recommend you call `reset` on logout even if you don't expect users to share a computer.**
-
-You can do that like so:
-
-<MultiLanguage selector="tabs">
-
-```javascript
-posthog.reset()
-```
-
-```iOS
-posthog.reset()
-```
-```Android
-PostHog.with(this)
-       .reset();
-```
-
-</MultiLanguage>
-
-If you _also_ want to reset the `device_id` so that the device will be considered a new device in future events, you can pass `true` as an argument:
-
-<MultiLanguage selector="tabs">
-
-```javascript
-posthog.reset(true)
-```
-
-```iOS
-posthog.reset(true)
-```
-
-```Android
-PostHog.with(this)
-       .reset(true);
-```
-</MultiLanguage>
+<JSIdentifyBestPractices3/>
 
 ### 4. Setting user properties
 
-You'll notice that one of the parameters in the `identify` method is a `properties` object. This enables you to set [user properties](/docs/getting-started/user-properties). Whenever possible, we recommend passing in all user properties you have available each time you call identify, as this will ensure that their user profile on PostHog is up to date.
-
-User properties can also be set using a `capture` and not only with `identify`. See our [user properties docs](/docs/getting-started/user-properties) for more details on how to work with them and best practices.
+<JSIdentifyBestPractices4/>
 
 ## Alias: Assigning multiple distinct IDs to the same user 
```

Comment:
```suggestion
## Assigning multiple distinct IDs to the same user with `alias`
```

Reply:
I had this originally, but decided to have `Alias:` in the beginning since I think it will be better search experience